releases/release-1.31/release-notes/maps/pr-122499-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 122499
+releasenote:
+  text: StatefulSet autodelete will respect controlling owners on PVC claims as described
```

```suggestion
  text: StatefulSet autodelete respected controlling owners on PVC claims as described
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653161205